scanspec/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
What is this file for? I thought only the .vscode in the root was used?

Reply:
This file shouldnt be here, its a duplicate. I have removed it.